src/subcommands/deployments.ts

```diff
@@ -41,23 +52,47 @@ Or just show the details of a specific deployment, of any project, using --id. T
 
     deployctl deployments show --id=p63c39ck5feg --next
 
+## List
+
+The "deployments list" subcommand is used to list the deployments of a project. 
+
+The simples form of the command will list the first 20 deployments of the project you are currently
+in (project will be picked up from the config file):
+
+    deployctl deployments list
+
+You can list the rest of the deployments using --page, --next or --prev:
+
+    deployctl deployments list --page=2
```

Comment:
How would `--next` or `--prev` option determine the "base" page? Is `--page=2` different from `--next=2`?

Reply:
It's the same. I deliverately focused the example only on using `--page` as I wonder too if it's of any value having `--next` and `--prev` in this subcommand. However, the arguments are there anyway, so the harm of their presence is not significant, and I don't want to rule out the possibility that combining these flags could be useful to somebody.

Reply:
I personally don't see benefits of suppoorting `--next` and `--prev`, although I do agree that the presence of these won't be of any harm. (I'd prefer, though, to keep it simple, support orthogonal features unless we have strong need)
As long as `--page=N` suffices, how about introducing `--page` only for now, and adding `--next` and `--prev` later if they turn out to be needed?

Reply:
I agree that for some users, seeing these options might stir up some confusion. What if I remove them from the docs completely, but still leave their implementation around? The code is quite simple

Reply:
> What if I remove them from the docs completely, but still leave their implementation around?

Sounds like a reasonable compromise. Let's do that way and assume that those options may be subject to breaking changes.